test/fixtures/desks.yml

```diff
@@ -5,6 +5,11 @@
 # below each fixture, per the syntax in the comments below
 #
 desk1:
-  name: desk1
+  name: Occupied Desk
   grouping: grouping1
   occupied: true
+
+desk2:
```

Comment:
My bad, I should've explained better. I meant changing the name of the fixture itself, so it results in:
```yaml
occupied_desk:
  grouping: grouping1
  occupied: true

unoccupied_desk:
  grouping: grouping1
  occupied: false
```